libretro_core_options.h

```diff
@@ -443,13 +443,13 @@ struct retro_core_option_v2_definition option_defs_us[] = {
       BEETLE_OPT(gun_cursor),
       "Gun Cursor",
       NULL,
-      "Select the gun cursor to be displayed on screen while using the 'Guncon/G-Con 45' and 'Justifier' input device types. When disabled, crosshairs are always hidden.",
+      "Choose the gun cursor to be displayed on screen while using the 'Guncon/G-Con 45' and 'Justifier' input device types. When disabled, crosshairs are always hidden.",
```

Comment:
I think some people would not identify a cursor as a "crosshair" or "sight", as commonly written in other emulators. What do you think?

Reply:
Let's stick to crosshair then? Seems like the more common term.

Reply:
Okay then.